onnxruntime/python/_pybind_state.py

```diff
@@ -5,9 +5,23 @@
 
 import os
 import platform
+import sys
 import warnings
 import onnxruntime.capi._ld_preload  # noqa: F401
 
+# Python 3.8 (and later) on Windows doesn't search system PATH when loading DLLs,
+# so CUDA location needs to be specified explicitly.
+if platform.system() == "Windows" and sys.version_info >= (3, 8):
+    CUDA_VERSION = "10.2"
```

Comment:
Even though all onnxruntime's prebuilt packages were built with CUDA 10.2, but at source level onnxruntime is compatible with multiple CUDA versions(including 10.2 and 11.0). I hope this can be handled better.

Reply:
Tracked by https://github.com/microsoft/onnxruntime/issues/5962.